views/roles.html

```diff
@@ -17,27 +17,27 @@ <h1>Roles</h1>
         </div>
     </header>
     <main class="container mt-4">
-        <section id="roles" class="mt-4">
+        <section id="bandRoles" class="mt-4">
             <h2>Kainos Job Roles</h2>
-            {% if roles and roles.length > 0 %}
+            {% if bandRoles and bandRoles.length > 0 %}
             <table class="table">
                 <thead>
                     <tr>
                         <th scope="col">Role Name</th>
                         <th scope="col">Capability</th>
                         <th scope="col">Responsibilities</th>
+                        <th scope="col">Sharepoint Link</th>
                         <th scope="col">Role Level</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {% for role in roles %}
+                    {% for bandRole in bandRoles %}
                     <tr>
-                        <td><a href="job-role-spec/{{ role.roleName }}">{{role.roleName}}</a></td>
-                        <td>{{role.capabilityName}}</td>
-                        <td>{{role.responsibilities}}</td>
-                        {% for band in bands %}
-                        <td>{{band.bandLevel}}</td>
-                        {% endfor %}
+                        <td>{{bandRole.roleName}}</td>
+                        <td>{{bandRole.specification}}</td>
+                        <td>{{bandRole.responsibilities}}</td>
+                        <td><a href="bandRole.sharepointLink">Sharepoint</a></td>
+                        <td>{{bandRole.bandLevel}}</td>
```

Comment:
I think the band name would be more interesting to display that the number of the level. i.e. "trainee" would look better than "7"